packages/apps/kadena-docs/src/pages/_app.tsx

```diff
@@ -1,17 +1,46 @@
-import type { AppProps } from 'next/app';
+import { baseGlobalStyles, globalCss } from '@kadena/react-components';
+
+import { Footer, Header } from '@/components';
+import { getLayout } from '@/utils';
+import { AppProps } from 'next/app';
 import Head from 'next/head';
 import React, { ComponentType } from 'react';
 
+const GlobalStyles = globalCss({
+  ...baseGlobalStyles,
+  body: {
+    background: '$background',
+  },
+});
+GlobalStyles();
+
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export default function App({ Component, pageProps }: AppProps): JSX.Element {
+  const { markdoc } = pageProps;
+
+  let title, description;
+  let layoutType = 'default';
+  if (markdoc !== undefined) {
```

Comment:
should we not just check on `!markdoc` in case of `null`, also it's shorter :P

Reply:
i tried again. but the linting rule prevents this, in this case